src/scheduler/workers/unpublish/dayBefore/sendDayBeforeUnpublishPostReminder.ts

```diff
@@ -0,0 +1,67 @@
+import { schedulerLogger } from "scheduler/logger";
+import { NotifyClient, RequestError } from "notifications-node-client";
+import { NOTIFY } from "server/config";
+import { Meta } from "../types";
+import { dayBeforePostReminderPersonalisation } from "./dailyReminderPersonalisation";
+import {AuditEvent, List} from "@prisma/client";
+import {recordListItemEvent} from "server/models/audit";
+
+const template = NOTIFY.templates.unpublishNotice.postOneDay;
+
+const logger = schedulerLogger.child({ method: "sendUnpublishEmail", template });
+const notifyClient = new NotifyClient(NOTIFY.apiKey);
```

Comment:
Looks like we have a third similar email file. Maybe worth adding a refactoring ticket reduce duplication (via a utils file or something) in these sorts of files?